src/scripts/add-implementation.js

```diff
@@ -1,11 +1,13 @@
 import AppController from "../models/AppController";
 
-export default function addImplementation({ contractName, contractAlias, packageFileName = undefined }) {
-  if (contractName === undefined) throw new Error('Must provide a contract name')
-  if (!contractAlias) contractAlias = contractName
+export default function addImplementation({ contractsData, packageFileName = undefined }) {
+  if (contractsData.length === 0) throw new Error('Must provide at least one contract name')
 
   const appController = new AppController(packageFileName)
-  appController.validateImplementation(contractName)
-  appController.addImplementation(contractAlias, contractName)
-  appController.writePackage()
+
+  contractsData.forEach(({ name, alias }) => {
+    appController.validateImplementation(name)
+    appController.addImplementation(alias, name)
+    appController.writePackage()
```

Comment:
Let's write the package file after the for-loop, so the command is transactional, and does not make any changes if one contract fails